### PR TITLE
chore: migrate project to schema 49

### PR DESCRIPTION
**Pull request by yajuaryarudderstack**
## Description of the change

> Project migrated to schema 49

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: models/inputs.yaml [hunk 1]
```diff
@@ -20,15 +20,12 @@ inputs:
         - select: "user_id"
           type: user_id
           entity: user
-          to_default_stitcher: true
```
Comment (shekhar-rudder) [inline models/inputs.yaml:23]: .DS_Store file should not be part of this PR

File: models/profiles-ml.yaml [hunk 2]
```diff
@@ -46,17 +42,14 @@ models:
               features:
                 - name: *percentile_name_7_days
                   description: 'Percentile of churn score. Higher the percentile, higher the probability of churn'
-
-      <<: *feature_meta_data_7_days
-
+      !!merge <<: *feature_meta_data_7_days
```
Comment (shekhar-rudder) [inline models/profiles-ml.yaml:45]: Where is `!!merge` coming from ?

File: models/inputs.yaml (hunk 1 above)
Comment (yajuaryarudderstack) [reply models/inputs.yaml:23]: Removed.

File: models/profiles-ml.yaml (hunk 2 above)
Comment (yajuaryarudderstack) [reply models/profiles-ml.yaml:45]: @sprksh can you please check?

Comment (yajuaryarudderstack) [reply models/profiles-ml.yaml:45]: This came via auto migrate. @shekhar-rudder

File: models/inputs.yaml (hunk 1 above)
Comment (shekhar-rudder) [reply models/inputs.yaml:23]: I still see it in the PR.

Comment (yajuaryarudderstack) [reply models/inputs.yaml:23]: @shekhar-rudder you can see that the file is now in bin: https://github.com/rudderlabs/rudderstack-profiles-base-features/pull/29/commits/f6690abff00c84e6b766193890ba7e26469cb813

Comment (shekhar-rudder) [reply models/inputs.yaml:23]: My bad. I didn't notice that it is an already committed file

File: models/profiles-ml.yaml (hunk 2 above)
Comment (sprksh) [reply models/profiles-ml.yaml:45]: https://blogs.perl.org/users/tinita/2019/05/reusing-data-with-yaml-anchors-aliases-and-merge-keys.html
So  `<<: ` is the shorthand for `!!merge <<:`
So while yaml migration, its converting to the explicit form.

**Review by shekhar-rudder — APPROVED** [review]
